autogluon/task/tabular_prediction/predictor.py

```diff
@@ -386,6 +388,43 @@ def feature_importance(self, model=None, dataset=None, features=None, raw=True,
 
         return self._learner.get_feature_importance(model=model, X=dataset, features=features, raw=raw, subsample_size=subsample_size, silent=silent)
 
+    def refit_full(self, model=None):
+        """
+        Optimizes a model's inference time by collapsing bagged ensembles into a single model fit on all of the training data.
```

Comment:
What do you think about extending this function (in separate PR) to also be applicable to non-bagged models trained via a standard train/val split? (FULL just refits on train+val in this case).

I'm not sure we want user to have to learn too many refit_full variants for different predictors generated via different task.fit() args.

Reply:
Clarified in documentation